test/models/notification_test.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+# test the Notification model
+class NotificationTest < UnitTestCase
+  def test_summary
```

Comment:
What about notifications is this testing?  It seems like it's just validating the fixtures.

Reply:
(Grr. I accidentally clobbered my explanation.)
- I don't think the fixtures say or imply that `notification.summary` for a Name includes the display_name.
- I don't think the fixtures say or imply that the `summary` for a bad flavor is the expected string.
Background: I was trying to add some coverage to stop Coveralls from failing. The second assertion covers an allegedly "newly-missed" line. The first assertion is there for balance; it looks silly to just cover the unhappy path.

Questions: 
- Is there a better way to test `summary`?
- Are you OK with the above explanation.

Reply:
Got it!  I didn't realize summary was a method and I didn't realize it was to make Coveralls happy.